modules/otel-collector-components.adoc

```diff
@@ -557,7 +557,38 @@ config: |
 <2> The default exporter when the attribute value is not present in the table in the next section.
 <3> The table that defines which values are to be routed to which exporters.
 
-You can optionally create an `attribute_source` configuratiion, which defines where to look for the attribute in `from_attribute`. The allowed value is `context` to search the context, which includes the HTTP headers, or `resource` to search the resource attributes.
+You can optionally create an `attribute_source` configuration, which defines where to look for the attribute in `from_attribute`. The allowed value is `context` to search the context, which includes the HTTP headers, or `resource` to search the resource attributes.
+
+[id="cumulativetodelta-processor_{context}"]
+=== Cumulative to Delta processor
+
+The Cumulative to Delta processor is currently a link:https://access.redhat.com/support/offerings/techpreview[Technology Preview] feature only.
+
+This processor converts monotonic, cumulative-sum, and histogram metrics to monotonic delta metrics.
+
+[NOTE]
+====
+This processor does not convert non-monotonic sums and exponential histograms.
+====
+
+.OpenTelemetry Collector custom resource with an enabled Cumulative to Delta processor
+[source,yaml]
+----
+config: |
+  processors:
+    cumulativetodelta:
+      include: # <1>
+        match_type: strict
+        metrics:
+        - first_metric
+        - second_metric
+      exclude: # <2>
+        match_type: regexp
+        metrics:
+        - "metric_.*"
+----
+<1> Optional: Configures which metrics to include. When omitted, all metrics are converted to delta metrics. The `match_type` field supports two values, `strict` and `regexp`, and defines if the metric must be an exact match with any metric or regular expression that is listed in `metrics` field. The `metrics` field defines specific metric names or regular expressions to convert to delta metrics.
```

Comment:
> When omitted, all metrics are converted to delta metrics

This is not 100% correct, if `include` is omitted but `exclude` is defined, it won't convert all metrics, but instead all metrics except for the ones in `exclude`.


```suggestion
<1> Optional: Configures which metrics to include. When omitted, all metrics (except for metrics listed in `exclude`) are converted to delta metrics. The `match_type` field supports two values, `strict` and `regexp`, and defines if the metric must be an exact match with any metric or regular expression that is listed in `metrics` field. The `metrics` field defines specific metric names or regular expressions to convert to delta metrics.
```

Feel free to update my suggestion, probably there's a better way to formulate it.

Reply:
Thank you for the clarification. I updated it to the following:

```
When omitted, all metrics, except for those listed in the `exclude` field, are converted to delta metrics.
```